enhance: update config.edn template

Bad3r opened the pull request:
This PR is a replacement for #9034 after addressing feedback.

- [x] Replace 'repo' w/ graph or graph folder, respectively.
- [x] Each configuration options documentation must start with a capital letter and end with a dot to maintain uniformity.
- [x] Maintain uniformity by using the same language and documentation style where applicable
- [x] Validate spelling and grammar
- [x] Uncomment UI configurable options and set them to the default value - At present, when a configuration option like `:ui/show-brackets? true` is commented out in the template; Logseq will add a new line to the bottom of the configuration template if the user toggles it from the UI instead of updating the existing variable.
- [x] define default values where applicable
- [x] Add warnings about potential issues
- [x] Validate the completeness of the template
- [x] Ensure that the `L` in Logseq is always capitalized
- [x] Use "Logseq" instead of "the app" when referring to Logseq.
- [x] Update docstring
- [x] Review Default Values

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;; Available options:
+ ;; - "Markdown" (default)
+ ;; - "Org"
+ :preferred-format "Markdown"
```
source:  
 

```clojure
(defn get-preferred-format
  ([]
   (get-preferred-format (get-current-repo)))
  ([repo-url]
   (keyword
     (or
       (when-let [fmt (:preferred-format (get-config repo-url))]
         (string/lower-case (name fmt)))

       (get-in @state [:me :preferred_format] "markdown")))))
```

Bad3r replied:
@logseq-cldwalker should all be commented out? my only concern is that when users modify options that are available via the settings UI it will create a duplicate entry at the bottom of the file. 

Possible options as I see it:
A. comment out all options after setting the default value
B. uncomment all options w/ default value
C. uncomment-only options configurable via settings UI.

Bad3r replied:
It's worth noting that once the user changes the setting from the UI a new line will be added at the bottom of the file. The format selection is available in the settings UI:  
 

![](https://user-images.githubusercontent.com/25513724/235818811-ebd68509-48cb-490c-b9dd-9b9efa16ad95.png)

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;; Available options:
+ ;; - ":now" for NOW/LATER style (default)
+ ;; - ":todo" for TODO/DOING style
  :preferred-workflow :now
```
souce:  
```cljs
(defn get-preferred-workflow
  []
  (keyword
    (or
      (when-let [workflow (:preferred-workflow (get-config))]
        (let [workflow (name workflow)]
          (if (util/safe-re-find #"now|NOW" workflow)
            :now
            :todo)))
      (get-in @state [:me :preferred_workflow] :now))))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; :journal/page-title-format "EEE, do MMM yyyy"
+ ;; Set a custom date format for the journal page title.
+ ;; Default value: "MMM do, yyyy"
+ :journal/page-title-format "MMM do, yyyy"
```
The code uses \`MMM do, yyyy\` as the default if none is specified. I do think that \`EEE, do MMM yyyy\` is better, and I think this option has been set like this for a long time.

https://github.com/logseq/logseq/blob/a5e31128a6366df002488203406684f78d80c7e3/deps/graph-parser/src/logseq/graph_parser/config.cljs#L76-82

@cnrpman, should the default value be updated to \`EEE, do MMM yyyy\`

andelf replied:
This deviates from the default behavior and is a personal preference.
At least, this should be marked as BREAKING CHANGE.

Bad3r replied:
@andelf which is the default behaviour?   
A. `MMM do, yyyy`  
  
B. `EEE, do MMM yyyy`  
  
According to [https://github.com/logseq/logseq/blob/master/deps/graph-parser/src/logseq/graph_parser/config.cljs#L76-82](https://github.com/logseq/logseq/blob/master/deps/graph-parser/src/logseq/graph_parser/config.cljs#L76-82)  
```cljs  
(defn get-date-formatter  
 \[config\]  
 (or  
  (:journal/page-title-format config)  
  ;; for compatibility  
  (:date-formatter config)  
  "MMM do, yyyy"))

```
  
  
the default value is B. Which one should be used as the default?

Bad3r replied:
@andelf I appreciate that it's a matter of personal preference, and I'm just curious about which one you would recommend as the default value in the default config template, as well as the expected value if none is provided in the source code.

Bad3r replied:
Also btw I just added an example to the docstring.

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;;   To show old journal files in the app, manually rename the files in the
+ ;;   journal directory to match the new format.
+ ;; Default value: "yyyy_MM_dd"
+ :journal/file-name-format "yyyy_MM_dd"
```
```clojure
(def default-journal-title-formatter (tf/formatter "yyyy_MM_dd"))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; Default is true, you can also toggle this via setting page
+ ;; Enable tooltip preview on hover.
+ ;; Default value: true
  :ui/enable-tooltip? true
```
```clojure
(defn enable-tooltip?
  []
  (if (mobile?)
    false
    (get (sub-config) :ui/enable-tooltip? true)))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; :ui/show-brackets? true
+ ;; Display brackets [[]] around page references.
+ ;; Default value: true
+ :ui/show-brackets? true
```
```clojure
;; Enable by default
(defn show-brackets?
  []
  (not (false? (:ui/show-brackets? (sub-config)))))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; Enable showing the body of blocks when referencing them.
+ ;; Display the complete block content when referencing.
+ ;; Default value: false
  :ui/show-full-blocks? false
```
```clojure
(defn toggle-show-full-blocks! []
  (let [show-full-blocks? (state/show-full-blocks?)]
    (set-config! :ui/show-full-blocks? (not show-full-blocks?))))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; Expand block references automatically when zoom-in
+ ;; Automatically expand block references when zooming in.
+ ;; Default value: true
  :ui/auto-expand-block-refs? true
```
```clojure
  "Default config for a repo-specific, user config"
  {:feature/enable-search-remove-accents? true
   :default-arweave-gateway "https://arweave.net"
   :ui/auto-expand-block-refs? true
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; Enable Block timestamp
+ ;; Enable Block timestamps.
+ ;; Default value: false
  :feature/enable-block-timestamps? false
```
\`\`\`cljs  
(defn enable-block-timestamps?  
 \[\]  
 (true? (:feature/enable-block-timestamps? (sub-config))))  
\`\`\`

Bad3r replied:
```clojure
(defn enable-block-timestamps?
  []
  (true? (:feature/enable-block-timestamps? (sub-config))))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;; Disable accent marks when searching.
+ ;; After changing this setting, rebuild the search index by pressing (^C ^S).
+ ;; Default value: true
  :feature/enable-search-remove-accents? true
```
```clojure
  "Default config for a repo-specific, user config"
  {:feature/enable-search-remove-accents? true
   :default-arweave-gateway "https://arweave.net"
   :ui/auto-expand-block-refs? true
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; :feature/enable-journals? true
+ ;; Enable journals.
+ ;; Default value: true
+ :feature/enable-journals? true
```
```clojure
(defn enable-journals?
  ([]
   (enable-journals? (get-current-repo)))
  ([repo]
   (not (false? (:feature/enable-journals? (sub-config repo))))))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; :feature/enable-flashcards? true
+ ;; Enable flashcards.
+ ;; Default value: true
+ :feature/enable-flashcards? true
```
```clojure
(rum/defc flashcards-enabled-switcher
  [enable-flashcards?]
  (ui/toggle enable-flashcards?
             (fn []
               (let [value (not enable-flashcards?)]
                 (config-handler/set-config! :feature/enable-flashcards? value)))
             true))
```

Bad3r replied:
```clojure
(rum/defc flashcards-enabled-switcher
  [enable-flashcards?]
  (ui/toggle enable-flashcards?
             (fn []
               (let [value (not enable-flashcards?)]
                 (config-handler/set-config! :feature/enable-flashcards? value)))
             true))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; :feature/enable-whiteboards? true
+ ;; Enable whiteboards.
+ ;; Default value: true
+ :feature/enable-whiteboards? true
```
```clojure
(rum/defc whiteboards-enabled-switcher
  [enabled?]
  (ui/toggle enabled?
             (fn []
               (let [value (not enabled?)]
                 (config-handler/set-config! :feature/enable-whiteboards? value)))
             true))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; :feature/disable-scheduled-and-deadline-query? true
+ ;; Disable the journal's built-in 'Scheduled tasks and deadlines' query.
+ ;; Default value: false
+ :feature/disable-scheduled-and-deadline-query? false
```
```clojure
(defn scheduled-deadlines-disabled?
  []
  (true? (:feature/disable-scheduled-and-deadline-query? (sub-config))))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;; Display all scheduled tasks and deadlines for the next 14 days:
  ;; :scheduled/future-days 14
+ ;; Default value: 7
+ :scheduled/future-days 7
```
```clojure
(defn get-scheduled-future-days
  []
  (let [days (:scheduled/future-days (get-config))]
    (or (when (int? days) days) 7)))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;; Available options:
+ ;;  - integer from 0 to 6 (Monday to Sunday) 
+ ;; Default value: 6 (Sunday)
  :start-of-week 6
```
```clojure
(defn get-start-of-week
  []
  (or (:start-of-week (get-config))
      (get-in @state [:me :settings :start-of-week])
      6))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; If `:sidebar` is not set, sidebar will be hidden
- ;; Example:
- ;; 1. Setup page "Changelog" as home page and "Contents" in sidebar
+ :arweave/gateway "https://arweave.net"
```
```clojure
:default-arweave-gateway "https://arweave.net"
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;;  - `:four-spaces` as four spaces
+ ;;  - `:two-spaces` as two spaces
+ ;; - `:tab` as a tab character (default)
+ :export/bullet-indentation :tab
```
```clojure
(defn get-export-bullet-indentation
  []
  (case (get (get-config) :export/bullet-indentation :tab)
    :eight-spaces
    "        "
    :four-spaces
    "    "
    :two-spaces
    "  "
    :tab
    "\t"))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;; Publish all pages within the Graph
+ ;; regardless of whether individual pages have been marked as public.
+ ;; Default value: false
+ :publishing/all-pages-public? false
```
```clojure
(defn get-export-bullet-indentation
  []
  (case (get (get-config) :export/bullet-indentation :tab)
    :eight-spaces
    "        "
    :four-spaces
    "    "
    :two-spaces
    "  "
    :tab
    "\t"))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+
+ ;; Set the default location for storing notes.
+ ;; Default value: "pages"
+ :pages-directory "pages"
```
```clojure
(defn get-pages-directory
  []
  (or
    (when-let [repo (get-current-repo)]
      (:pages-directory (get-config repo)))
    "pages"))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; :pages-directory "your-directory"
+ ;; Set the default location for storing journals.
+ ;; Default value: "journals"
+ :journals-directory "journals"
```
```clojure
(defn get-journals-directory
  []
  (or
    (when-let [repo (get-current-repo)]
      (:journals-directory (get-config repo)))
    "journals"))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
- ;; :journals-directory "your-directory"
+ ;; Set the default location for storing whiteboards.
+ ;; If not specified, whiteboards will be stored in the 'whiteboards' subdirectory.
+ :whiteboards-directory "whiteboards"
```
```clojure
(defn get-whiteboards-directory
  []
  (or
   (when-let [repo (get-current-repo)]
     (:whiteboards-directory (get-config repo)))
   "whiteboards"))
```

Bad3r commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;; Enabling this option converts
+ ;; [[Grant Ideas]] to [[file:./grant_ideas.org][Grant Ideas]] for org-mode.
+ ;; For more information, visit https://github.com/logseq/logseq/issues/672
+ :org-mode/insert-file-link? false
```
```clojure
(defn org-mode-file-link?
  [repo]
  (:org-mode/insert-file-link? (get-config repo)))
```

Bad3r commented on templates/config.edn:
```diff
@@ -146,33 +176,38 @@
+ ;; Configure the behavior of pressing Enter in document mode.
+ ;; if set to true, pressing Enter will create a new block.
+ ;; Default value: false
  :shortcut/doc-mode-enter-for-new-block? false
```
```clojure
(defn doc-mode-enter-for-new-line?
  []
  (and (document-mode?)
       (not (:shortcut/doc-mode-enter-for-new-block? (get-config)))))
```

Bad3r commented on templates/config.edn:
```diff
@@ -146,33 +176,38 @@
- ;; or editable for performance.
+ ;; Limit the searchable and editable block content length for performance
+ ;; Default value: 10000
  :block/content-max-length 10000
```
```clojure
(defn block-content-max-length
  [repo]
  (or (:block/content-max-length (sub-config repo)) 10000))
```

Bad3r commented on templates/config.edn:
```diff
@@ -146,33 +176,38 @@
- ;; Whether to show command doc on hover
+ ;; Display command documentation on hover.
+ ;; Default value: true
  :ui/show-command-doc? true
```
```clojure
(defn show-command-doc?
  []
  (get (sub-config) :ui/show-command-doc? true))
```

Bad3r commented on templates/config.edn:
```diff
@@ -146,33 +176,38 @@
 
- ;; Whether to show empty bullets for non-document mode (the default mode)
+ ;; Display empty bullet points.
  :ui/show-empty-bullets? false
```
the default value is nil or false

https://github.com/logseq/logseq/blob/a5e31128a6366df002488203406684f78d80c7e3/src/main/frontend/components/block.cljs?plain=1#LL1770C1-L1774C16
```cljs
       (cond
         (and (or (mobile-util/native-platform?)
                  (:ui/show-empty-bullets? (state/get-config)))
              (not doc-mode?))
         bullet
```

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
  ;; - block title
  ;;   block content
+ ;; Default value: false
  :outliner/block-title-collapse-enabled? false
```
default value is nil or false
```cljs
    (let [title-collapse-enabled? (:outliner/block-title-collapse-enabled? (state/get-config))]
      (when (and (not block-ref-with-title?)
                 (seq body)
                 (or (not title-collapse-enabled?)
                     (and title-collapse-enabled?
                          (or (not collapsed?)
                              (some? (mldoc/extract-first-query-from-ast body))))))
        [:div.block-body
         ;; TODO: consistent id instead of the idx (since it could be changed later)
         (let [body (block/trim-break-lines! (:block/body block))]
           (for [[idx child] (medley/indexed body)]
             (when-let [block (markup-element-cp config child)]
               (rum/with-key (block-child block)
```

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
- :ref/linked-references-collapsed-threshold 50
+ ;; Configure the threshold for linked references before collapsing.
+ ;; Default value: 100
+ :ref/linked-references-collapsed-threshold 100
```
```clojure
(defn get-linked-references-collapsed-threshold
  []
  (or
    (when-let [value (:ref/linked-references-collapsed-threshold (get-config))]
      (when (integer? value)
        value))
    100))
```

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
+ ;; With the default value of level 2, block b will be collapsed.
+ ;; If the level's value is set to 3, block c will be collapsed.
+ ;; Default value: 2
  :ref/default-open-blocks-level 2
```
```clojure
(defn get-ref-open-blocks-level
  []
  (or
    (when-let [value (:ref/default-open-blocks-level (get-config))]
      (when (integer? value)
        value))
    2))
```

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
+ ;; - Float between 0 and 1
+ ;; higher values result in faster changes to the next review interval.
+ ;; Default value: 0.5
+ :srs/learning-fraction 0.5
```
```clojure
(def learning-fraction-default
  "any number between 0 and 1 (the greater it is the faster the changes of the OF matrix)"
  0.5)
```

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
- :property-pages/enabled? true
+ ;; Create a page for all properties.
+ ;; default value: false
+ :property-pages/enabled? false
```
default value is nil or false
```cljs
what is the default value?
        property-pages-enabled? (contains? #{true nil} (:property-pages/enabled? user-config))]
```

logseq-cldwalker replied:
Value changed

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
  ;; Enable logical outdenting
- ;; :editor/logical-outdenting? true
+ ;; default value: false
+  :editor/logical-outdenting? false
```
````clojure
Default value is nil or false
```cljs
(defn logical-outdenting?
  []
  (:editor/logical-outdenting? (sub-config)))
```
````

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
- ;;  :media "[[quick capture]] **{time}**: {url}"}
+  :quick-capture-templates
+  {:text "[[quick capture]] **{time}**: {text} — {url}"
+   :media "[[quick capture]] **{time}**: {url}"}
```
@andelf is it ok to add an em dash (—) to the default text template to separate the text and URL? In the previously suggested template, the word 'from' was used instead. The default value does not contain a separator between the text and the URL. I strongly believe that adding an em dash (—) would improve the default template. It is a small change, but I think it's a good quality change.

Please let me know if I should revert it.

```cljs
        template (get-in (state/get-config)
                         [:quick-capture-templates :text]
                         "**{time}** [[quick capture]]: {text} — {url}")
```

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
+  :quick-capture-options
+  {:insert-today? false            ;; Default value: true
+   :redirect-page? false          ;; Default value: false
+   :default-page "quick capture"} ;; Default value: "quick capture" page
```
```clojure
        insert-today? (get-in (state/get-config)
                              [:quick-capture-options :insert-today?]
                              false)
        redirect-page? (get-in (state/get-config)
                               [:quick-capture-options :redirect-page?]
                               false)
```

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
+   :block-ref?       true        ;; Default value: true
+   :page-ref?        true        ;; Default value: true
+   :properties?      true        ;; Default value: true
+   :list?            true}       ;; Default value: true
```
```clojure
(def default-settings
  {:admonition&src?  true
   :markup?          false
   :block-ref?       true
   :page-ref?        true
   :properties?      true
   :list?            true})
```

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
- ;; :journal/file-name-format "yyyy_MM_dd"
-
- }
+  :file/name-format :triple-lowbar}
```
the default value is `legacy` for compatibility reasons. All new graphs should use `:triple-lowbar`

hezhizhen replied:
One space was added accidentally. It's better to remove it.

And I think it's better add `:legacy` to available options list.

Bad3r commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
+ ;; Mobile photo upload configuration.
+ :mobile/photo
+ {:allow-editing? true
+  :quality        80}
```
```clojure
  (-> (.getPhoto Camera (clj->js
                         {:allowEditing (get-in
                                         (state/get-config)
                                         [:mobile/photo :allow-editing?])
                          :quality (get-in (state/get-config)
                                           [:mobile/photo :quality] 80)
                          :saveToGallery true
                          :resultType (.-Base64 CameraResultType)}))
```

Bad3r commented:
Hello,

I have closed the old PR (#9034) and opened this one to make it easier to review and reduce clutter.

Please review this PR when you get a chance. Thank you for all your help and feedback \\:D 

CC:

@cnrpman, @hezhizhen, @phoenixeliot

andelf commented on templates/config.edn:
```diff
@@ -1,137 +1,167 @@
+ ;; Set the preferred workflow style.
+ ;; Available options:
+ ;; - ":now" for NOW/LATER style (default)
+ ;; - ":todo" for TODO/DOING style
```
Better not to use double quotes to denote keywords

andelf reviewed (changes requested):
It seems that these changes will overwrite any global config.
PTAL @logseq-cldwalker

hezhizhen commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
+ ;;   re-index them on all clients after synchronization.
+ ;;   Incorrect handling may result in messy page titles.
+ ;; Available options:
+ ;;   -:triple-lowbar (default)
```
need a space between dash and options

hezhizhen commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
+ ;; - default-page:   The default page to capture to if insert-today? is false.
+  :quick-capture-options
+  {:insert-today? false            ;; Default value: true
+   :redirect-page? false          ;; Default value: false
```
aligning comments would be better

hezhizhen commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
 
  ;; Enable logical outdenting
- ;; :editor/logical-outdenting? true
+ ;; default value: false
```
Default?

hezhizhen commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
+  :enabled-in-timestamped-blocks false
+  ; limit logbook to minutes, eliminating seconds. (Default value: false)
+  :with-second-support? false
+  ; display logbook in all blocks after time-tracking. (Default value: false)
```
two ; ?

hezhizhen commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
- ;; the initial interval after the first successful review of a card (default 4)
- ;; :srs/initial-interval 4
+ ;; Set the initial interval after the first successful review of a card.
+ ;; default value: 4
```
Default?

hezhizhen commented on templates/config.edn:
```diff
@@ -207,145 +242,179 @@
- ;; (multiple lines) can be collapsed too. For example:
+ ;;  ["js" "Javascript"]
+ ;;  ["md" "Markdown"]
+ ;  ]
```
add ;

hezhizhen commented on templates/config.edn:
```diff
@@ -207,145 +246,179 @@
+   :media "[[quick capture]] **{time}**: {url}"}
+
+ ;; Quick capture options.
+ ;; - insert-today?:  Insert the capture at the end of today's journal page.
```
`- :insert-today? Insert ...` should be better?

hezhizhen commented on templates/config.edn:
```diff
@@ -207,145 +246,179 @@
- ;; any number between 0 and 1 (the greater it is the faster the changes of the next-interval of card reviews) (default 0.5)
- ;; :srs/learning-fraction 0.5
+ ;; Set flashcards interval.
+ ;; Default value: 0.5
```
duplicated line.

hezhizhen replied:
duplicated line.

logseq-cldwalker commented on templates/config.edn:
```diff
@@ -1,137 +1,171 @@
+ :journal/page-title-format "MMM do, yyyy"
+
+ ;; Specify the journal filename format using a valid date format string.
+ ;; !Warning:
```
👍

logseq-cldwalker commented on templates/config.edn:
```diff
@@ -1,137 +1,171 @@
+ ;; the 'scheduled tasks and deadlines' query.
  ;; Example usage:
- ;; Display all scheduled and deadline blocks for the next 14 days
+ ;; Display all scheduled tasks and deadlines for the next 14 days:
```
We should revert wording here. The new wording is incorrect as this functionality isn't limited to tasks

logseq-cldwalker commented on templates/config.edn:
```diff
@@ -1,137 +1,171 @@
- ;; https://github.com/logseq/awesome-logseq#css-themes
- ;; Example:
- ;; :custom-css-url "@import url('https://cdn.jsdelivr.net/gh/dracula/logseq@master/custom.css');"
+ ;; Import a custom CSS file.
```
file should be url

logseq-cldwalker commented on templates/config.edn:
```diff
@@ -1,137 +1,171 @@
- ;; Specify a custom js import
- ;; This option take precedence over your local `logseq/custom.js` file
- ;; :custom-js-url ""
+ ;; Import a custom JS file.
```
file should be url

logseq-cldwalker commented on templates/config.edn:
```diff
@@ -1,137 +1,171 @@
- ;; 4. use `false` to disable particular shortcut
- ;; 5. you can define multiple bindings for one action, eg `["ctrl+j" "down"]`
- ;; full list of configurable shortcuts are available below:
+ ;; 1. + indicates simultaneous key presses, e.g., ctrl+shift+a.
```
Would be helpful to keep backticks as it reads better

logseq-cldwalker commented on templates/config.edn:
```diff
@@ -146,33 +180,38 @@
 
- ;; Block content larger than `block/content-max-length` will not be searchable
- ;; or editable for performance.
+ ;; Limit the searchable and editable block content length for performance
```
The previous comment explains functionality better

logseq-cldwalker reviewed (changes requested):
@Bad3r Hi. Most of these changes seem reasonable. Left some feedback to address

Bad3r commented:
@logseq-cldwalker, @andelf  Thank you for promptly reviewing my pull request. I have addressed the feedback you provided, with the exception of the issue concerning the commenting out of config options. I am awaiting your response on that matter for further clarification, as mentioned in above.  
  
Thanks again for your help and insight.

logseq-cldwalker commented on templates/config.edn:
```diff
@@ -207,145 +245,178 @@
- :ref/linked-references-collapsed-threshold 50
+ ;; Configure the threshold for linked references before collapsing.
+ ;; Default value: 100
+ :ref/linked-references-collapsed-threshold 100
```
Value changed

logseq-cldwalker reviewed (approved):
@Bad3r Thanks for making config.edn template more consistent! 👍 🚢  In a followup commit I'll address the things that weren't addressed and derisk the PR to ensure no default values were changed
